Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -681,12 +714,10 @@ - (NSURL *)fileURLForBookIndentifier:(NSString *const)identifier
 - (NSURL *)fileURLForBookIndentifier:(NSString *const)identifier account:(NSString * const)account
 {
   if(!identifier) return nil;
-  
```

Comment:
can you leave this comment and move it to your new `pathExtensionForBook:`  method, unless it's wrong? it seems quite useful. It could be on its documentation for example.

Reply:
moved it to `pathExtensionForBook`, it's not wrong but it seems other libraries work fine with `.epub` file name extensions